src/rcpp_parse_chromatograms.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+#include "external_libraries.hpp"
+#include "xml_utils.h"
+#include <string>
+#include <Rcpp.h>
+
+
+// [[Rcpp::export]]
+Rcpp::List rcpp_parse_chromatograms(std::string file_path, Rcpp::IntegerVector index = NA_INTEGER) {
```

Comment:
I prefer to pass STL objects by (const) reference, e.g. `const std::string &file_path`.

(although, admittedly, for strings it won't matter much probably. Mainly to be consistent).